src/WebJobs.Script.WebHost/packages.config

```diff
@@ -2,27 +2,33 @@
-  <package id="Microsoft.AspNet.WebHooks.Receivers.Pusher" version="1.2.0-rc1c" targetFramework="net46" />
-  <package id="Microsoft.AspNet.WebHooks.Receivers.Salesforce" version="1.2.0-rc1c" targetFramework="net46" />
-  <package id="Microsoft.AspNet.WebHooks.Receivers.Slack" version="1.2.0-rc1c" targetFramework="net46" />
-  <package id="Microsoft.AspNet.WebHooks.Receivers.Stripe" version="1.2.0-rc1c" targetFramework="net46" />
-  <package id="Microsoft.AspNet.WebHooks.Receivers.Trello" version="1.2.0-rc1c" targetFramework="net46" />
-  <package id="Microsoft.AspNet.WebHooks.Receivers.WordPress" version="1.2.0-rc1c" targetFramework="net46" />
-  <package id="Microsoft.Azure.ApiHub.Sdk" version="0.6.7-alpha" targetFramework="net46" />
+  <package id="Microsoft.AspNet.WebHooks.Common" version="1.2.0-rc1" targetFramework="net46" />
+  <package id="Microsoft.AspNet.WebHooks.Receivers" version="1.2.0-beta6" targetFramework="net45" />
+  <package id="Microsoft.AspNet.WebHooks.Receivers.Azure" version="1.2.0-beta6" targetFramework="net45" />
+  <package id="Microsoft.AspNet.WebHooks.Receivers.Bitbucket" version="1.2.0-beta6" targetFramework="net45" />
+  <package id="Microsoft.AspNet.WebHooks.Receivers.Dropbox" version="1.2.0-beta6" targetFramework="net45" />
+  <package id="Microsoft.AspNet.WebHooks.Receivers.DynamicsCRM" version="1.2.0-beta6" targetFramework="net45" />
+  <package id="Microsoft.AspNet.WebHooks.Receivers.Generic" version="1.2.0-beta6" targetFramework="net45" />
+  <package id="Microsoft.AspNet.WebHooks.Receivers.GitHub" version="1.2.0-beta6" targetFramework="net45" />
+  <package id="Microsoft.AspNet.WebHooks.Receivers.Instagram" version="1.2.0-beta6" targetFramework="net45" />
+  <package id="Microsoft.AspNet.WebHooks.Receivers.MailChimp" version="1.2.0-beta6" targetFramework="net45" />
+  <package id="Microsoft.AspNet.WebHooks.Receivers.Pusher" version="1.2.0-beta6" targetFramework="net45" />
+  <package id="Microsoft.AspNet.WebHooks.Receivers.Salesforce" version="1.2.0-beta6" targetFramework="net45" />
+  <package id="Microsoft.AspNet.WebHooks.Receivers.Slack" version="1.2.0-beta6" targetFramework="net45" />
+  <package id="Microsoft.AspNet.WebHooks.Receivers.Stripe" version="1.2.0-beta6" targetFramework="net45" />
+  <package id="Microsoft.AspNet.WebHooks.Receivers.Trello" version="1.2.0-beta6" targetFramework="net45" />
+  <package id="Microsoft.AspNet.WebHooks.Receivers.WordPress" version="1.2.0-beta6" targetFramework="net45" />
+  <package id="Microsoft.AspNetCore.Http.Abstractions" version="1.0.0-rc2-final" targetFramework="net46" />
+  <package id="Microsoft.AspNetCore.Http.Extensions" version="1.0.0-rc2-final" targetFramework="net46" />
+  <package id="Microsoft.AspNetCore.Http.Features" version="1.0.0-rc2-final" targetFramework="net46" />
+  <package id="Microsoft.Azure.ApiHub.Sdk" version="0.6.6-alpha" targetFramework="net46" />
```

Comment:
Not sure if this was a conflict issue, but we're downgrading many of these packages, which we don't want to do.